e2e/test/scenarios/admin/databases/database-prompt-banner.cy.spec.js

```diff
@@ -89,3 +159,14 @@ describeWithSnowplow(
     });
   },
 );
+
+const visitUrl = url => {
```

Comment:
copied from https://github.com/metabase/metabase/blob/d679380fe1e243b552efa9009bba789a5787ea06/e2e/test/scenarios/embedding/embedding-full-app.cy.spec.js#L315-L324

The reason I don't put these test in other existing embedding tests is because I think it'd be easier to extend/modify the behavior of this feature if every test case are co-located in a single place.